src/api.ts

```diff
@@ -150,23 +151,32 @@ export function getType(value: IAnyStateTreeNode): MSTAnyComplexType | IAnyCompl
   return value[$type];
 }
 
-export function getEnv<Env = any>(value: IAnyStateTreeNode): Env {
+/** @hidden */
+export function getContext(value: IAnyStateTreeNode): InstantiateContext | null {
```

Comment:
nit: since it's used beyond instantiation now, I'd be down for giving `InstantiateContext` a more generic name.